gammapy/estimators/flux_map.py

```diff
@@ -455,102 +470,119 @@ def eflux_ref(self):
     @property
     def dnde(self):
         """Return differential flux (dnde) SED values."""
-        return self.norm * self.dnde_ref
+        return self._filter_convergence_failure(self.norm * self.dnde_ref)
```

Comment:
if it has been filtered for `norm` it is useless to redo it for `flux` quantities

Reply:
This has been fixed along the whole code